src/time_travel/datetime_patcher.py

```diff
@@ -10,39 +12,40 @@ class DatetimePatcher(object):
     For now patching only the @today function.
     """
     
-    def __init__(self, start_time=0):
+    def __init__(self, clock):
         """Create the patch.
         
         @start_time is time in seconds since the epoch.
         """
-        self.timestamp = start_time
+        self.clock = clock
         
-        self.real_datetime = datetime_lib.datetime.fromtimestamp
+        self.real_fromtimestamp = datetime_lib.datetime.fromtimestamp
         
         self.datetime = mock.Mock(wraps=datetime_lib.datetime)
         self.datetime.mock_add_spec(
             ["today", "now", "utcnow", "fromtimestamp"])
```

Comment:
There's a problem with mock's {{wraps}} that it's not wrapping class methods. After adding the class methods to the That was the mock's spec they are wrapped correctly. I might find a better solution later, but that one was quick and easy for now.